code/modules/hydroponics/hydroponics.dm

```diff
@@ -648,7 +649,7 @@
 			to_chat(user, "<span class='notice'>This plant must be harvestable in order to be grafted.</span>")
 			return
 		else if(myseed.grafted)
-			to_chat(user, "<span class='notice'>This plant has already been grafted.</span>")
+			to_chat(user, "<span class='notice'>This plant has already had something sheared or grafted off from it.</span>")

```

Comment:
I don't see where shears set this to true.

Also will need to update documentation of this variable if not change the name now that it also considers shearing.

Reply:
I had it so that gene shearing was a once/per life like grafts, but I decided against it by going with just causing plant_health damage instead.